SickSangHae/View/HistoryView.swift

```diff
@@ -9,120 +9,218 @@ import SwiftUI
 
 struct HistoryView: View {
     
-    let array = [
-        "당근", "계란30구", "과자", "팝콘", "요거트", "당당치킨", "친환경양파", "깐마늘",
-        "맛있는우유", "월드콘", "돼지고기 목살", "깻잎", "딸기잼", "머스타드 소스", "된장",
-        "고추장", "어묵", "아무거나"
-    ]
+    @StateObject var coreDataViewModel = CoreDataViewModel()
+    
+    @State var isMovingSegmentedTab : Bool = true
+    
+    let array = ["Apple", "Banana", "Orange", "Pineapple", "Grapes", "Watermelon"]
```

Comment:
```suggestion
    let array: [String] = ["Apple", "Banana", "Orange", "Pineapple", "Grapes", "Watermelon"]
```

배열의 경우에는 타입 명시가 되어있지 않으면 컴파일에서 시간을 많이 잡아먹는 요인이됩니당 ㅠㅠ
다른 타입은 생략하더라도 배열은 꼭 명시해주면 좋아요 !

Reply:
감사합니다!